x-pack/legacy/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -56,6 +57,18 @@ export class HeadlessChromiumDriverFactory {
 
   type = BROWSER_TYPE;
 
+  async screenshotPage() {
+    if (!this.page) {
+      throw new Error(`Page creation hasn't happened yet`);
+    }
+
```

Comment:
We could do `page.pdf` here instead, which renders a format that users at least expect. Downside here is that it's generally a bit more expensive than screenshots